pkg/status/cluster_status.pb.go

```diff
@@ -1,17 +1,17 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
```

Comment:
@evan-cz `make generate` was failing, so I deleted `pkg/status/cluster_status.pb.go` and regenerated. is that the correct set of steps?